build.yaml

```diff
@@ -33,7 +33,10 @@ steps:
        valueFrom: default_ns.name
    public: false
    secrets:
+    - database-server-config
+    - gcr-push-service-account-key
```

Comment:
So these are OK for now, since we only test authorized changes and protect the logs, but eventually we're going to want to fully isolate the tests from production.  You don't need to change anything, just something to keep in mind.